src/service.ts

```diff
@@ -22,3 +23,17 @@ export const getDates = async () => {
         alert(err);
     }
 };
+
+export const getStackedChart = async (options: {
+    contact: string;
+    type: string;
+}): Promise<StackedChartData[] | undefined> => {
+    try {
+        const response = await axios.get(`${backendUrl}/stackedchart`, {
```

Comment:
use hyphen between words in url https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

Reply:
will change this in my next PR